src/main/archetype/ui.content/src/main/content/jcr_root/conf/__confFolderName__/settings/wcm/policies/.content.xml

```diff
@@ -46,6 +46,206 @@
                     </form-container>
                 </container>
             </form>
+            <content jcr:primaryType="nt:unstructured">
+                <carousel jcr:primaryType="nt:unstructured">
+                    <policy_1558955398911
```

Comment:
Instead of auto generated policy number, would it be helpful to define user-friendly policy name?

Reply:
fixed